examples/SmartShop/contracts/BuyerContract.aes

```diff
@@ -0,0 +1,31 @@
+contract Remote =
+  function received_item : () => bool
+  function seller_contract_balance : () => int
+  function check_item_status : () => string
+
+  function check_courier_status : () => string
+  function check_courier_location : () => string
+  function check_courier_timestamp : () => int
+  
+contract Buyer =
+  function deposit_to_seller_contract(price : int, key : address) : bool =
+    Chain.spend(key, price)
```

Comment:
maybe on buyer init should initialize with seller address?

Reply:
The idea of this example is to show how to make remote calls and very basic seller-buyer behavior implementation.